ADSDeploy/config.py

```diff
@@ -9,7 +9,7 @@
 # Configuration of the pipeline; if you start 'vagrant up rabbitmq' 
 # container, the port is localhost:6672 - but for production, you 
 # want to point to the ADSImport pipeline 
-RABBITMQ_URL = 'amqp://guest:guest@127.0.0.1:6672/?' \
+RABBITMQ_URL = 'amqp://guest:guest@172.17.0.1:6672/?' \
```

Comment:
Wasn't meant to commit. Was purely because I was working in the container and looking at the functional tests.